packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -15,37 +15,42 @@ const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClic
           Courses
         </div>
         {
-                    autoCompleteHits.filter(hit => hit.content_type === 'course')
-                      .slice(0, MAX_NUM_SUGGESTIONS)
-                      .map((hit) => (
-                        <SearchSuggestionItem
-                          url={getLinkToCourse(hit)}
-                          /* eslint-disable-next-line no-underscore-dangle */
-                          highlightedTitle={hit._highlightResult.title.value}
-                          authoringOrganization={hit.key && hit.key.split('+')[0]}
-                          title={hit.title}
-                        />
-                      ))
-                  }
+          autoCompleteHits.filter(hit => hit.content_type === 'course')
+            .slice(0, MAX_NUM_SUGGESTIONS)
+            .map((hit) => (
+              <SearchSuggestionItem
+                key={hit.title}
+                url={getLinkToCourse(hit)}
+                /* eslint-disable-next-line no-underscore-dangle */
+                highlightedTitle={hit._highlightResult.title.value}
+                authoringOrganization={hit.key && hit.key.split('+')[0]}
+                title={hit.title}
+              />
+            ))
+        }
       </div>
       <div>
         <div className="mb-2 mt-5 ml-2 font-weight-bold suggestions-section">
           Programs
         </div>
         {
-                    autoCompleteHits.filter(hit => hit.content_type === 'program')
-                      .slice(0, MAX_NUM_SUGGESTIONS)
-                      .map((hit) => (
-                        <SearchSuggestionItem
-                          url={getLinkToProgram(hit)}
-                          /* eslint-disable-next-line no-underscore-dangle */
-                          highlightedTitle={hit._highlightResult.title.value}
-                          authoringOrganization={hit.authoring_organizations && hit.authoring_organizations[0].key}
-                          title={hit.title}
-                          programType={hit.program_type}
-                        />
-                      ))
-                  }
+          autoCompleteHits.filter(hit => hit.content_type === 'program')
+            .slice(0, MAX_NUM_SUGGESTIONS)
+            .map((hit) => {
+              const orgFound = hit.authoring_organizations && hit.authoring_organizations.length > 0;
```

Comment:
nit: `const orgFound = hit.authoring_organizations?.length > 0;`

And maybe `hasAuthoringOrg` vs. `orgFound` to be a bit more descriptive?